src/transformer/mockDefiner/mockDefiner.ts

```diff
@@ -91,6 +91,12 @@ export class MockDefiner {
         return this._getCallGetFactory(key);
     }
 
+    public getMockFactoryByKey(key: string): ts.Expression {
```

Comment:
you could change getMockFactory to call this one

Reply:
Done! thank you!!!!